lib/srv/db/db_test.go

```diff
@@ -172,27 +176,142 @@ func TestDatabaseAccess(t *testing.T) {
 	}
 }
 
+// TestMySQLAccess verifies access scenarios to a MySQL database based
+// on the configured RBAC rules.
+func TestMySQLAccess(t *testing.T) {
```

Comment:
Is it possible to consolidate Postgres and MySQL tests? Maybe even follow the suite pattern we have for the backends? Would make writing the implementation of additional databases easier.

Reply:
The tests cases are actually a little different for Postgres and MySQL, so I'm not immediately sure how to consolidate them further - they already share the common setup for example via setupTestContext method.